CitationStyleLanguagePlugin.inc.php

```diff
@@ -428,10 +482,10 @@ public function getCitation($request, $article, $citationStyle = 'apa', $issue =
 
         // DOI
         if ($issue && $issue->getPublished()) {
```

Comment:
Checking the issue here might not make sense in OPS. DOIs will exist even if there is no issue. @ewhanson when do you think that a DOI should be shown in a citation? Whenever it is set on the publication? Only when it has a particular status?

Reply:
I think it should be shown whenever it is set on the publication. If it's added as part of the publication's metadata, I believe it should be included in the citation. Also, I don't think it should be status dependent, as this may unnecessarily hide DOIs when a status changes or a DOI registration agency is disabled.

Reply:
It has been updated.